src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -786,598 +452,215 @@ public enum CallOfLegends implements LogicCardInfo {
-              damage 0
-            }
-          }
-          move "Dual Splash", {
-            text "Choose 2 of your opponent’s Pokémon. This attack does 30 damage to each of them."
-            energyCost W, W, C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
-      case CHIKORITA_53:
-        return basic (this, hp:HP050, type:GRASS, retreatCost:1) {
-          weakness R
-          resistance W, MINUS20
-          move "Nap", {
-            text "Remove 1 damage counter from Chikorita."
-            energyCost C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-          move "Reckless Charge", {
-            text "30 damage. Chikorita does 10 damage to itself."
-            energyCost G, C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
-      case CLEFAIRY_54:
-        return basic (this, hp:HP050, type:COLORLESS, retreatCost:1) {
-          weakness F
-          move "Minimize", {
-            text "."
-            energyCost C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-          move "Slap", {
-            text "10 damage. "
-            energyCost C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
-      case CYNDAQUIL_55:
-        return basic (this, hp:HP040, type:FIRE, retreatCost:1) {
-          weakness W
-          move "Fireworks", {
-            text "20 damage. Energy attached to Cyndaquil."
-            energyCost R, R
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
-      case EEVEE_56:
-        return basic (this, hp:HP050, type:COLORLESS, retreatCost:1) {
-          weakness F
-          move "Tackle", {
-            text "10 damage. "
+          resistance W, MINUS20
+          move "Grind", {
+            text "20× damage. Does 20 damage times the amount of Energy attached to Tangrowth."
             energyCost C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 20*self.cards.energyCount(C)
             }
           }
-          move "Bite", {
-            text "20 damage. "
-            energyCost C, C
+          move "Plow Over", {
+            text "30 damage. Flip a coin. If heads, the Defending Pokémon is now Paralyzed. If tails, put 1 Energy card attached to the Defending Pokémon in the Lost Zone."
+            energyCost G, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 30
+              flip 1, {
+                apply PARALYZED
+              }, {
+                defending.cards.filterByType(ENERGY).select("Choose an energy to put into the Lost Zone").moveTo(opp.lostZone)
```

Comment:
I prefer using the filter for CardList `select`s on a Pokémon's cards. That way you can see all of the cards currently attached in case it could inform your decision.

Reply:
In my experience, filtering before selection increases the clarity of the effect as it only shows your valid choices. I prefer to use the filter parameter of select for effects that have you choose one of n cards that meets a criteria such as team up jirachi.

Reply:
I can see that, however other cards attached to the Pokémon may change which card you choose. The text string should provide clarify on what should be selected, and the other cards can help inform the decision IMO.